manifests/role/hathitrust/dev/app_host.pp

```diff
@@ -9,4 +9,5 @@
 class nebula::role::hathitrust::dev::app_host {
   include nebula::role::hathitrust::dev
   include nebula::profile::named_instances
+  include nebula::profile::ruby
```

Comment:
Now buzz and grog can be regular app hosts, because ht-web-testing no longer depends mysteriously on this role.